login/apps/login/src/lib/server/cookie.ts

```diff
@@ -55,10 +55,21 @@ export async function createSessionAndUpdateCookie(command: {
   const _headers = await headers();
   const { serviceUrl } = getServiceUrlFromHeaders(_headers);
 
+  let sessionLifetime = command.lifetime;
+
+  if (!sessionLifetime) {
+    console.warn("No session lifetime provided, using default of 24 hours.");
```

Comment:
Shouldn't this be either an error or an infinite lifetime?

Reply:
when you enter the loginname we create the session now with 24h lifetime, instead of infinite.
This will be overridden with the lifetime of the method used later though.
This function will be invoked without a value though as there is none for this use case

Reply:
24h is enough for loginnames without having them verified anytime. That's where this fallback comes in.